src/markdown-pages/docs/managers/zos3270terminal-manager.md

```diff
@@ -42,6 +46,12 @@ import dev.galasa.zos3270.spi.Colour;
 import dev.galasa.zos3270.spi.Highlight;
 ```
 
+To use the (experimental) feature for saving PNG representations of terminal screens to the RAS in a test, import the following component into the test:
+
+```
+import dev.galasa.zos3270.internal.properties.TerminalOutput;
+```
+
```

Comment:
We can remove this bit since the `zos3270.terminal.output` property is set in the CPS.

Reply:
will do